Build/Build.cs

```diff
@@ -70,9 +68,18 @@ class Build : NukeBuild
     AbsolutePath TestResultsDirectory => RootDirectory / "TestResults";
 
     string SemVer;
-    string YarnCli => ToolPathResolver.GetPackageExecutable("Yarn.MSBuild", "yarn.js", "1.22.19");
+
+    Target Informations => _ => _
+        .OnlyWhenDynamic(() => !IsLocalBuild)
+        .Executes(() =>
+        {
+            Information(BranchSpec);
+            Information(BuildNumber);
+            Information(PullRequestBase);
+        });
```

Comment:
Reason for this new target: I often had to look in the build logs (especially during the last PRs regarding the NUKE pipeline) to see the value of various variables.

Now... Put them in this target if you care.